develop/dev-guide-gui-navicat.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Connect to TiDB with Navicat
+summary: Learn how to connect to TiDB using Navicat.
+---
+
+# Connect to TiDB with Navicat
+
+TiDB is a MySQL-compatible database, and [Navicat](https://www.navicat.com) is a GUI tool set for database users. This tutorial uses the [Navicat for MySQL](https://www.navicat.com/en/products/navicat-for-mysql) tool to connect to TiDB.
+
+> **Warning:**
+>
+> Navicat is not full supported TiDB. Navicat can connect to TiDB, because of TiDB is a MySQL-compatible database. So the Navicat will treat TiDB as a MySQL. But there some errors might be occered when you are using it. Besides, the Navicat is not free. If it is possible, recommand you use some other free GUI tools, they supported TiDB officially:
```

Comment:
```suggestion
> Navicat doesn't fully support TiDB. Navicat can connect to TiDB, because of TiDB being a MySQL-compatible database. So Navicat will treat TiDB as a MySQL. But there some errors might occur when you are using it. Besides, the Navicat is not free. It is recommended to use some other free GUI tools that supported TiDB officially:
```